ucx.spec

```diff
@@ -1,19 +1,24 @@
-### RPM external ucx 1.12.0
-Source: https://github.com/openucx/%{n}/releases/download/v%{realversion}/%{n}-%{realversion}.tar.gz
+### RPM external ucx 1.12.1
+Source: https://github.com/openucx/%{n}/archive/refs/tags/v%{realversion}.tar.gz
 BuildRequires: autotools
+Requires: cuda gdrcopy
 Requires: numactl
-Requires: gdrcopy cuda
 Requires: rdma-core
+Requires: rocm
```

Comment:
@fwyzard , this will package `rocm` for all archs. I think we should include and configure `rocm` only for x86_64 archs

Reply:
you are right, of course.